src/library/scala/collection/mutable/ImmutableBuilder.scala

```diff
@@ -18,7 +18,7 @@ package mutable
 /**
   * Reusable builder for immutable collections
   */
-abstract class ImmutableBuilder[-A, C](empty: C)
+abstract class ImmutableBuilder[-A, C <: IterableOnce[_]](empty: C)
```

Comment:
is this change change still necessary? I'm not clear enough on the hierarchy to be sure

Reply:
Yes, it is. There's nothing we could do with a completely unconstrained `C`